docs/benchmarking.md

```diff
@@ -0,0 +1,43 @@
+# Benchmarking Workflows
+
+<!-- used for displaying doctree when selecting file -->
+```{toctree}
+:maxdepth: 4
+```
+
+## Introduction to Benchmarking with Cytosnake
+
+Cytosnake is a Python package known for it  collection of workflows designed for analyzing image-based profiles.
+To ensure these workflows perform at their best, it's essential to conduct benchmarking.
+Benchmarking allows us to assess and optimize the efficiency and reliability of Cytosnake's image analysis processes.
+This tutorial will guide you through the benchmarking process, helping you gauge and improve the performance of these image-based profiling workflows.
+Whether you're a developer seeking to enhance Cytosnake's functionalities or a user interested in its performance, this guide will equip you with the necessary knowledge.
```

Comment:
Will the context of development for CytoSnake have any expectations when it comes to OS, resources, etc?

Reply:
Hey Dave, unfortunately I do not have an answer to this question right now. This could be due to how new this implementation is. My inital thought would be that it should work on all OS's and the Benchmarking repo should provide more information in terms or how the resources are used. 

In respect to the OS question, I don't have the unit/functional tests to back that up yet.

Reply:
Thanks @axiomcura ! It might be useful to consider outlining what is expected to happen on how this influences benchmarking for CytoSnake. Specifically here I think about how MacOS vs Linux vs Windows might opt to parallelize or thread the work. It could be useful to know how and where the steps are accomplished at an abstract level (how steps sequence and/or parallelize) so as to inform users and also increase benchmarking understanding. This is more of a suggestion than a requirement for review, and I trust your best judgement here.